pysteps/blending/steps.py

```diff
@@ -1451,10 +1456,52 @@ def worker(j):
                         # forecast outside the radar domain. Therefore, fill these
                         # areas with the "..._mod_only" blended forecasts, consisting
                         # of the NWP and noise components.
+
                         nan_indices = np.isnan(R_f_new)
-                        R_f_new[nan_indices] = R_f_new_mod_only[nan_indices]
-                        nan_indices = np.isnan(R_pm_blended)
-                        R_pm_blended[nan_indices] = R_pm_blended_mod_only[nan_indices]
+                        if smooth_radar_mask_range != 0:
```

Comment:
What happens if the value is negative?

Reply:
A  raise ValueError("max_padding_size_in_px must be greater than or equal to 0.") is thrown in the bledning.util.compute_smooth_dilated_mask( ... ) function